proto/arkeo/arkeo/events.proto

```diff
@@ -10,10 +10,14 @@ import "arkeo/arkeo/keeper.proto";
 option go_package = "github.com/arkeonetwork/arkeo/x/arkeo/types";
 
 message EventBondProvider {
-  string creator = 1;
-  string provider = 2 [ (gogoproto.casttype) = "github.com/arkeonetwork/arkeo/common.PubKey" ];
-  string service = 3;
-  string bond = 4 [
+  string provider = 1 [ (gogoproto.casttype) = "github.com/arkeonetwork/arkeo/common.PubKey" ];
+  string service = 2;
+  string bondRel = 3 [
```

Comment:
I must have missed when these variables got introduced, but thoughts on a better / clearer name?  Not really related to this PR, but just caught my eye.